config/routes.go

```diff
@@ -66,22 +66,23 @@ func setupAPI(e *echo.Echo) {
 	d.POST("/", controllers.CreateDatacenterHandler)
 	d.PUT("/:project", controllers.UpdateDatacenterHandler)
 	d.DELETE("/:project", controllers.DeleteDatacenterHandler)
-	d.DELETE("/:project/envs/:env", controllers.DeleteServiceHandler)
-	d.POST("/:project/envs/:env/reset/", controllers.ResetServiceHandler)
-	d.DELETE("/:project/envs/:env/force/", controllers.ForceServiceDeletionHandler)
-	d.PUT("/:project/envs/:env/", controllers.UpdateServiceHandler)
-	d.POST("/:project/envs/:env/sync/", controllers.SyncServiceHandler)
-	d.GET("/:project/envs/:env", controllers.GetServiceHandler)
-	d.GET("/:project/envs/:env/builds/", controllers.GetServiceBuildsHandler)
-	d.GET("/:project/envs/:env/builds/:build", controllers.GetServiceBuildHandler)
+	d.POST("/:project/envs/", controllers.CreateEnvHandler)
```

Comment:
I'm not sure, this should be under `/projects/:project/` as project is defined as part of the yaml, not the query. 
In the case we want to move it here, we will need to pre-process the yaml on the cli in order to extract the project name.